src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -181,6 +181,9 @@
     private static final Logger actionTrace = LogManager.getLogger("opendistro_security_action_trace");
     private static final DeprecationLogger deprecationLogger = DeprecationLogger.getLogger(OpenSearchSecurityPlugin.class);
 
+    public static final String LEGACY_OPENDISTRO_PREFIX = "/_opendistro/_security/";
```

Comment:
check for other places where ` "/_opendistro/_security/"` is used and replace them with constant (for example AuditLogging). The same for `"/_plugins/_security/"`.